site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+                      <li><a href="#" class="side-nav-item">Label</a></li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+              <div class="accordion-item">
+                <h4 class="accordion-header" id="exploreHeader">
+                  <button class="accordion-button collapsed side-nav-item fw-normal" type="button" data-bs-toggle="collapse" data-bs-target="#exploreCollapse" aria-expanded="false" aria-controls="exploreCollapse">
+                    Explore
+                  </button>
+                </h4>
+                <div id="exploreCollapse" class="accordion-collapse collapse" aria-labelledby="exploreHeader" data-bs-parent="#dashboardAccordion">
+                  <div class="accordion-body">
+                    <ul class="list-unstyled m-0">
+                      <li><a href="#" class="side-nav-item">Label</a></li>
+                      <li><a href="#" class="side-nav-item">Label</a></li>
+                      <li><a href="#" class="side-nav-item">Label</a></li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="accordion-item">
+        <h3 class="accordion-header" id="chartsHeader">
+          <button class="accordion-button collapsed side-nav-item fw-bold" type="button" data-bs-toggle="collapse" data-bs-target="#chartsCollapse" aria-expanded="false" aria-controls="chartsCollapse">
+            Charts
+          </button>
+        </h3>
+        <div id="chartsCollapse" class="accordion-collapse collapse" aria-labelledby="chartsHeader" data-bs-parent="#sideNavAccordion">
+          <div class="accordion-body">
+            <ul class="list-unstyled m-0">
+              <li><a href="#" class="side-nav-item">Label</a></li>
+              <li><a href="#" class="side-nav-item">Label</a></li>
+              <li><a href="#" class="side-nav-item">Label</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="accordion-item">
+        <a href="#" class="side-nav-item fw-bold">Financial</a>
+      </div>
+    </div>
+  </div>
+</nav>
+
+<div class="d-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+{{< /example >}}
+
+### With accordions
+
+Here is the `.side-nav-accordion` theme. It should be used with [`.side-nav-static`](#static) since it doesn't have any collapse behavior. The whole accordion should be scrollable.
+
+{{< example class="p-lg-0 side-nav-example d-block d-lg-flex" >}}
+<button class="btn btn-primary d-lg-none" data-bs-target="#sideNavOffcanvas" data-bs-toggle="offcanvas" aria-haspopup="true" aria-controls="sideNavOffcanvas" id="sideNavToggle">Show basic side navigation</button>
+
+<div class="offcanvas-lg offcanvas-start h-100" role="region" aria-labelledby="sideNavToggle" id="sideNavOffcanvas">
```

Comment:
Could be it confusing to use the `.h-100` utility class in some examples and to use a CSS "trick to remove `.h-100` inside the markup" in the others?

Reply:
This `.h-100` is outside the component so I couldn't replace it easily by CSS trick, I could study the impact inside offcanvas if needed.